### PR TITLE
add python image crawler demo

### PR DESCRIPTION
**Pull request by yinglixn**

### CODE REVIEW

File: image-crawler-python/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+#Fetch image from URL then upload to Aliyun OSS Example
+This example display how to fetch an image from remote source (URL) and then upload this image to a OSS bucket.
```
Comment (nerdyyatrice) [inline image-crawler-python/README.md]: This example demonstrates how to fetch an image from a remote web source and upload this image to an OSS bucket using the serverless framework with alibaba cloud as the provider.

File: image-crawler-python/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+## Setup
+
+* Install Serverless CLI v1.26.1+. You can get it by running `npm i -g serverless`.
+* An Aliyun account and open [Aliyun Function Compute](https://fc.console.aliyun.com) service
```
Comment (nerdyyatrice) [inline image-crawler-python/README.md]: Apply for an Alibaba cloud account and apply to use [Aliyun Function Compute](https://fc.console.aliyun.com) service

File: image-crawler-python/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+The structure of the project should look something like this:
+
+```
+├── index.py
+├── package.json
+└── serverless.yml
+```
+
+Install `serverless-aliyun-function-compute` plugin to your service from https://github.com/aliyun/serverless-aliyun-function-compute
```
Comment (nerdyyatrice) [inline image-crawler-python/README.md]: Install the latest  `serverless-aliyun-function-compute` plugin to your service

File: image-crawler-python/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+```sh
+serverless plugin install --name serverless-aliyun-function-compute
+```
+
+### Prepare credential file
```
Comment (nerdyyatrice) [inline image-crawler-python/README.md]: Prepare a credential file

File: image-crawler-python/index.py [hunk 5]
```diff
@@ -0,0 +1,53 @@
+# coding=utf-8
+'''
+    A demo of web image crawler that grabs all jpg files from a URL provided by the user and stores them in OSS.
```
Comment (nerdyyatrice) [inline image-crawler-python/index.py]: A demo of a web ...

File: image-crawler-python/package.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "image-crawler",
+  "version": "1.0.0",
+  "description": "Alibaba Cloud Function Compute Python Image Crawler for the Serverless Framework.",
+  "main": "index.py",
+  "license": "MIT",
+  "keywords": [
+    "serverless",
+    "aliyun"
```
Comment (nerdyyatrice) [inline image-crawler-python/package.json]: aliyun or alibaba cloud?

File: image-crawler-python/README.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+* You can find the `aliyun_access_key_secret` and `aliyun_access_key_id` from [AccessKey](https://ak-console.aliyun.com/?#/accesskey). You can also choose to create an Access Key or use sub-account Access Key.
+* You can find the `aliyun_account_id` from [Account](https://account-intl.console.aliyun.com/?#/secure).
+* After creating the `aliyun_credentials` file, please make sure to change the `credentials` field value in `serverless.yml` to the absolute file path.
+
+### Fill in your OSS bucket info
+
+* replace '<region>' with oss bucket region
+* replace '<bucket>' with oss bucket name
```
Comment (nerdyyatrice) [inline image-crawler-python/README.md]: in the index.py file

File: image-crawler-python/package.json (hunk 6 above)
Comment (yinglixn) [reply image-crawler-python/package.json]: "aliyun" is everywhere, will add "alibaba cloud" as well.

**Review by nerdyyatrice — APPROVED** [review]

**Pull request merged**